src/Model/Email/HoldEmail.php

```diff
@@ -26,9 +27,12 @@ public function getFromAddress()
      */
     public function getToAddress()
```

Comment:
@gkallenberg As per conversation, anything involving formatting Patron Info are now refactored into own functions in `HoldEmailData` class.